app/controllers/idv/review_controller.rb

```diff
@@ -62,6 +65,10 @@ def create
 
     private
 
+    def log_reproof_event
+      irs_attempts_api_tracker.idv_reproof
+    end
```

Comment:
We _could_ look at `user_session[:idv][:applicant]` here and get the data submitted, e.g.:

```
>> user_session[:idv][:applicant]
=> {"first_name"=>"FAKEY", "middle_name"=>nil, "last_name"=>"MCFAKERSON", "address1"=>"1 FAKE RD", "address2"=>nil, "city"=>"GREAT FALLS", "state"=>"MT", "zipcode"=>"59010", "dob"=>"1938-10-06", "state_id_number"=>"1111111111111", "state_id_jurisdiction"=>"ND", "state_id_type"=>"drivers_license", "state_id_expiration"=>"2099-12-31", "state_id_issued"=>"2019-12-31", "phone"=>"2015550123", "uuid"=>"b7e2770e-a0e7-4083-8476-0afe0d178a2b", "uuid_prefix"=>nil, "ssn"=>"666-11-2543"}
```

However, when a user reproofs, all of the normal "proofing" events are fired, so the information is already contained there and passing it in here seems redundant.

Since this event is essentially implementing the RISC event as an AttemptsEvent, and since the document information is already captured in other events, I decided to implement this like the RISC event that just states that the user has reproofed their account.